.github/workflows/tests.yml

```diff
@@ -53,6 +53,6 @@ jobs:
       if: matrix.config[1] == 'coverage'
       run: |
         pip install coveralls coverage-python-version
-        coveralls
+        coveralls --service=github
```

Comment:
AFAIU this is needed because of https://github.com/TheKevJames/coveralls-python/issues/252: sometimes coverage upload fails with 422 for obscure reasons, nobody understands why, and switching the service name sometimes helps.

And I see we had a failed coverage upload in git master earlier, and this branch fixes that.

Reply:
> sometimes helps

Indeed.

offtopic:
If you know another reason why those 422 are happening, please tell me :-/

In this one case it did not help - and I have no clue why. coveralls integration is a obscure thing
https://github.com/morepath/morepath_rest_dump_load/pull/9/checks?check_run_id=1690078143
https://github.com/morepath/morepath_rest_dump_load/blob/master/.github/workflows/main.yml